pkg/cmd/admin/router/router.go

```diff
@@ -487,11 +487,11 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 	if cfg.StatsPort > 0 {
 		port := kapi.ContainerPort{
 			Name:          "stats",
-			ContainerPort: cfg.StatsPort,
+			ContainerPort: int32(cfg.StatsPort),
```

Comment:
yeah, this is all over the place, I can go trough every change and figure out if I need to change the struct or just do a conversion.... doing this precisely will cause a significant delay for the rebase to land... I would rather create follow up issue for somebody (I think it will be me) to go trough all our types and use int32/64 consistently. wdyt?